src/Typesense/ITypesenseClient.cs

```diff
@@ -420,7 +421,6 @@ public interface ITypesenseClient
     /// <param name="collection">The collection name.</param>
     /// <param name="document">A documents of type T to update the filtered collection list with.</param>
     /// <param name="filter">The filter that is used to selected which documents that should be updated.</param>
-    /// <param name="filter">The filter that is used to selected which documents that should be updated.</param>
```

Comment:
Hi, I agree on this being deleted, it is possible to move this to another commit, since it is not related to this change.

Reply:
ok, removed and squashed